frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.styled.tsx

```diff
@@ -2,11 +2,18 @@ import styled from "@emotion/styled";
 
 import { Ellipsified } from "metabase/core/components/Ellipsified";
 import { Icon } from "metabase/core/components/Icon";
-import { color } from "metabase/lib/colors";
 import { space } from "metabase/styled-components/theme";
 
+// NIGHT-MODE TEXT HACK:
+// Our "fullscreen-night-text" className only supports one shade of text (white).
+// As a temporary fix, we create dark-mode analogs for "text-medium" and "text-light" using opacity:
+const detailsOpacity = 0.85; // approximates color("text-medium")
+const detailsNumberOpacity = 0.7; // approximates color("text-light")
+
```

Comment:
I do not quite understand why we are using opacity instead of directly setting the color here. Can you explain it to me?